test/python/test_nexus_multipath.py

```diff
@@ -268,6 +321,26 @@ def start_fio(create_nexus_dev):
     assert output.returncode == 0
 
 
+@pytest.fixture
```

Comment:
What is the purpose of these delays, could we add a comment to explain?

Reply:
Simply to wait for the kernel to notice NVMe path state changes. I'll add the comment.